conditional/__init__.py

```diff
@@ -1,14 +1,14 @@
 import os
 import subprocess
 from datetime import datetime
-from flask import Flask, redirect, request, render_template, g
-from flask_sqlalchemy import SQLAlchemy
-from flask_migrate import Migrate
+
```

Comment:
Move to the style fixes commit.

Also why is this being reordered (and some lines being deleted)?

Reply:
Based on: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Then that should be a separate commit from adding pyoidc support.